tests/potential/test_pair.py

```diff
@@ -334,6 +334,112 @@ def test_derivative(self):
         with self.assertRaises(ValueError):
             u = y._derivative(param='kapppa', r=r_input, epsilon=1.0, kappa=1.0)
 
+class test_LowBound(unittest.TestCase):
+    """Unit tests for relentless.potential.Depletion.LowBound"""
+
+    def test_init(self):
+        """Test creation from data"""
+        #create object dependent on scalars
+        w = relentless.potential.Depletion.LowBound(sigma_i=1.0, sigma_j=2.0)
+        self.assertAlmostEqual(w.value, 1.5)
+        self.assertCountEqual(w.params, ('sigma_i','sigma_j'))
+        self.assertDictEqual({p:v.value for p,v in w.depends},
+                             {'sigma_i':1.0, 'sigma_j':2.0})
+
+        #change parameter value
+        w.sigma_j.value = 4.0
+        self.assertAlmostEqual(w.value, 2.5)
+        self.assertCountEqual(w.params, ('sigma_i','sigma_j'))
+        self.assertDictEqual({p:v.value for p,v in w.depends},
+                             {'sigma_i':1.0, 'sigma_j':4.0})
+
+        #create object dependent on variables
+        a = relentless.DesignVariable(value=1.0)
+        b = relentless.DesignVariable(value=2.0)
+        w = relentless.potential.Depletion.LowBound(sigma_i=a, sigma_j=b)
+        self.assertAlmostEqual(w.value, 1.5)
+        self.assertCountEqual(w.params, ('sigma_i','sigma_j'))
+        self.assertDictEqual({p:v for p,v in w.depends},
+                             {'sigma_i':a, 'sigma_j':b})
+
+        #change parameter value
+        b.value = 4.0
+        self.assertAlmostEqual(w.value, 2.5)
+        self.assertCountEqual(w.params, ('sigma_i','sigma_j'))
+        self.assertDictEqual({p:v for p,v in w.depends},
+                             {'sigma_i':a, 'sigma_j':b})
+
+    def test_derivative(self):
+        """Test _derivative method"""
+        w = relentless.potential.Depletion.LowBound(sigma_i=1.0, sigma_j=2.0)
+        #calculate w.r.t. sigma_i
+        dw = w._derivative('sigma_i')
+        self.assertEqual(dw, 0.5)
+
+        #calculate w.r.t. sigma_j
+        dw = w._derivative('sigma_j')
+        self.assertEqual(dw, 0.5)
+
+        #invalid parameter calculation
+        with self.assertRaises(ValueError):
+            dw = w._derivative('sigma')
+
+class test_HighBound(unittest.TestCase):
```

Comment:
You can probably just put these tests as functions inside `test_Depletion`, so it's obvious what class they belong to.